### ISSUE TITLE
Unable to pass collation other than en_US.UTF8

### ISSUE THREAD

**Issue by aleksandrovpa** [terraform-google-modules/terraform-google-sql-db#433]
### TL;DR

For now it's impossible to create CloudSQL instance with collation other than en_US.UTF8 (missing one parameter in [sql_database_instance](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/sql_database_instance#collation))

### Expected behavior

Passing var db.collation and sql instance has default DB `postgres` with selected collation

### Observed behavior

SQL instance created with default database `postgres` with `en_US.UTF8` collation

### Terraform Configuration

```hcl
module "sql-db_postgresql" {
  source              = "GoogleCloudPlatform/sql-db/google//modules/postgresql"
  version             = "14.0.1"
  name                = var.name
  database_version    = var.database_version
  zone                = var.zone
  project_id          = var.project_id
  region              = var.region
  deletion_protection = var.deletion_protection
  // Master
  tier                 = var.tier
  disk_size            = var.disk_size
  disk_autoresize      = var.disk_autoresize
  availability_type    = var.availability_type
  ip_configuration     = var.ip_configuration
  backup_configuration = var.backup_configuration
  user_name            = var.user_name
  db_name              = var.db_name
  database_flags       = var.database_flags
  user_labels          = var.user_labels
  db_collation         = var.db_collation
  additional_databases = var.additional_databases
  additional_users     = var.additional_users
  // Read Replicas
  read_replicas                    = var.read_replicas
  read_replica_deletion_protection = var.read_replica_deletion_protection
}
```

### Terraform Version

```sh
Terraform v1.3.7
on darwin_arm64
```

### Additional information

Just add [here](https://github.com/terraform-google-modules/terraform-google-sql-db/blob/master/modules/postgresql/main.tf#L61) KV `collation                   = var.db_collation`
That's it

**Comment by aleksandrovpa** [comment]
i wanna send you PR, but it seems like i have no permissions(

**Comment by apeabody** [comment]
> i wanna send you PR, but it seems like i have no permissions(

Hi @aleksandrovpa - You should be able to fork this repo, make your commits to your fork, and when ready then open a PR from your fork's changes back to this repo.  Cheers!
